scripting/pugsetup/mapvote.sp

```diff
@@ -1,9 +1,53 @@
 #define RANDOM_MAP_VOTE "-1"  // must be in invalid index for array indexing
 
+char mapGroups[][] = {
+      "competitive.txt", "fun.txt", "all.txt"
+};
+
 /**
  * Map voting functions
  */
 public void CreateMapVote() {
+          StartMapVote2();
+}
+static void StartMapVote2() {
```

Comment:
This seems like a poor name for this function. Can you use a more descriptive name?